### ISSUE TITLE
pyhf xml2json crash

### ISSUE THREAD

**Issue by miochoa** [diana-hep/pyhf#396]
# Description

I installed pyhf 0.0.15 with Python 2 in a virtual environment and tested the following command:

```
pyhf xml2json config.xml > test.json
```

It produces the following crash:

```
  File "/data/users/miochoa/doubleTagger/gbb_new_fit/pyhf_workArea/pyhf/lib/python2.7/site-packages/pyhf/readxml.py", line 40, in import_root_histogram
    return h.numpy[0].tolist(), extract_error(h)
TypeError: 'instancemethod' object has no attribute '__getitem__' 
```

The list of packages used is:
```
pip list
Package        Version
-------------- -----------
awkward        0.8.1
cachetools     3.1.0
Click          7.0
functools32    3.2.3.post2
jsonschema     2.6.0
numpy          1.14.5
pip            19.0.1
pyhf           0.0.15
scipy          1.2.0
setuptools     40.7.1
six            1.12.0
tqdm           4.30.0
uproot         3.4.0
uproot-methods 0.4.1
wheel          0.32.3
```

The xml and root files tested can be found here: https://cernbox.cern.ch/index.php/s/cFO8U0YiOQjKjMu 

Thanks!
Inês

**Comment by matthewfeickert** [comment]
Hi @miochoa. Thanks for the bug report.

This has been fixed with release [`v0.0.16`](https://github.com/diana-hep/pyhf/releases/tag/v0.0.16) so if you do

```
pip install --upgrade pyhf[xmlimport]
```

that should fix things now.

The reason for your bug is a few things. First, the version that was on PyPI as `v0.0.15` doesn't include PR #228, PR #258 which add our dependencies on `six`, `jsonschema`, and `jsonpatch`. And then additionally it doesn't include PR #379 which was a fix to update an API breaking change from `uproot`. As these changes were introduced after `v0.0.15` was released they are not in PyPI but they were in `master`. They have now been included in `v0.0.16` and in the future we'll take steps to make sure that releases that worked in the past don't get broken by future changes in our dependencies.

I'm going to close this Issue now, but given the above if you update you should now be able to run the following with the example files you gave us.

```
pyhf xml2json config.xml > test.json
pyhf cls test.json 
```
and get the following output
```
{
    "CLs_exp": [
        4.4575064144162964e-08,
        1.8490768770125849e-06,
        6.454856267557041e-05,
        0.0016281355602771063,
        0.023527584634023567
    ],
    "CLs_obs": 0.5000161376615003
}
```

Let us know if you have any additional questions or problems on this.

**Comment by miochoa** [comment]
Hi Matthew,

The issue is fixed in v0.0.16 and I can reproduce that cls output. 

Thanks a lot for your quick help.

Best
Inês
